### PR TITLE
add digitalearth package 

### PR DESCRIPTION
**Pull request by MAfarrag**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by MAfarrag** [comment]
@conda-forge/python-help all chacks have passed here please review.

**Comment by MAfarrag** [comment]
@conda-forge/staged-recipes all chacks have passed here please review.

File: recipes/digitalearth/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    - geoplot >=0.5.1
+
+test:
+  imports:
+    - digitalearth
+
+about:
+  home: https://github.com/MAfarrag/digitalearth
+  license:  GPL-3.0-only
+  license_family: GPL
```
Comment (dopplershift) [inline recipes/digitalearth/meta.yaml:37]:
```suggestion
  license_family: GPL3
```

File: recipes/digitalearth/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    - loguru >=0.5.3
+    - cleopatra >=0.2.0
+    - geoplot >=0.5.1
+
+test:
+  imports:
+    - digitalearth
+
+about:
+  home: https://github.com/MAfarrag/digitalearth
```
Comment (dopplershift) [inline recipes/digitalearth/meta.yaml:35]:
```suggestion
  home: https://github.com/MAfarrag/Digital-Earth
```

File: recipes/digitalearth/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  license:  GPL-3.0-only
+  license_family: GPL
+  license_file: LICENSE.md
+  summary: Geo-spatial visualization package
+  description: |
+    Geo-spatial visualization package.
+  dev_url: https://github.com/MAfarrag/digitalearth
```
Comment (dopplershift) [inline recipes/digitalearth/meta.yaml:42]:
```suggestion
  dev_url: https://github.com/MAfarrag/Digital-Earth
```

File: recipes/digitalearth/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+  doc_url: https://digitalearth.readthedocs.io/
+  doc_source_url: https://github.com/MAfarrag/digitalearth/main/README.md
```
Comment (dopplershift) [inline recipes/digitalearth/meta.yaml:44]:
```suggestion
  doc_source_url: https://github.com/MAfarrag/Digital-Earth/main/README.md
```

File: recipes/digitalearth/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+  sha256: 17f933ead5a69b1ed76cbdfc46594e3771eb104606040c6fde197a3d08286859
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python  >=3.7.1,<3.10
```
Comment (dopplershift) [inline recipes/digitalearth/meta.yaml:19]:
```suggestion
    - python >=3.7.1,<3.10
```

File: recipes/digitalearth/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+    - pip >=21.3.1
+  run:
+    - python  >=3.7.1,<3.10
```
Comment (dopplershift) [inline recipes/digitalearth/meta.yaml:22]:
```suggestion
    - python >=3.7.1,<3.10
```

**Review by dopplershift — CHANGES_REQUESTED** [review]
A bunch of really minor things. The broken URLs is the main problem.

**Review by dopplershift — APPROVED** [review]

**Comment by MAfarrag** [comment]
@dopplershift thanks for your review, i have made the changes and now all the checks passed. Thanks

**Pull request merged**